### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Slouis07**
aboutuspage base example

### CODE REVIEW

File: aboutuspage.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  <header>
+      <nav class="navbar">
+          <div class="logo">
+            <img src = "birdlogo.png" alt="bird logo">
+          </div>
+          <div class="menu">
+              <ul>
+                  <li><a href="#">AboutUs</a></li>
+                  <li><a href="#">Terms & Conditions</a></li>
```
Comment (ItsJonasL) [inline aboutuspage.html:24]: ( li><a href="#">Terms &amp; Conditions</a></li ) 


use this instead the current code as that is triggering the linter to fail it.

File: aboutus.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
+
+
+body {
+    margin: 0;
+    padding: 0;
+    font-family: 'Arial', sans-serif;
```
Comment (ItsJonasL) [inline aboutus.css:6]: ya got to remove '' that is around Arial.

**Review by ItsJonasL — CHANGES_REQUESTED** [review]
please implement the changes because its failing on the linter. I have included the code that should be changed.

File: aboutuspage.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>replit</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <title> chirp website aboutus</title>
+</head>
+  
+
+<body>
```
Comment (DorisSanchez-Velasquez) [inline aboutuspage.html:13]: Awesome Work! Looks pretty good in terms of code! :)

Change Requested:
We're using React (.jsx) files in the main frontend code so a lot of this html and CSS can be transferred to the React files and prevent people from having to initiialize the <head> tags on every file. 

How To Implement Change:
Under the Pages folder, create another folder for AboutUs and create a .jsx file and a .module.css file and move all the code you in the <body> tag of this file into the .jsx file. Same thing with the .module.css file.  Checkout the Example.jsx file if you need help structuring out the .jsx file.

Reasoning:
React files are alot more flexible in terms of being able to implement functionality within the same files where you're inputting html structures. Essentially, React is a framework that combines everything you would put in a CSS, HTML, and JS file. :)

File: aboutus.css [hunk 4]
```diff
@@ -0,0 +1,132 @@
+
+
+body {
```
Comment (DorisSanchez-Velasquez) [inline aboutus.css:3]: Same change requested as the html file. Transfer all styling to a .module.css file within an AboutUs Folder. :)

**Review by DorisSanchez-Velasquez — CHANGES_REQUESTED** [review]
Great Work! Change requested in terms of making the html and css files adhere to React .jsx files.

**Comment by DerrickMac** [comment]
Closing pull request as updates to this were done on PR #33

**Pull request closed without merging**